ui/frontend/Output/Gist.tsx

```diff
@@ -54,16 +54,16 @@ class Copied extends React.PureComponent<CopiedProps, CopiedState> {
 
 const Links: React.SFC = () => {
   const codeUrl = useSelector(selectors.codeUrlSelector);
-  const gistUrl = useSelector((state: State) => state.output.gist.url);
-  const permalink = useSelector(selectors.permalinkSelector);
-  const urloUrl = useSelector(selectors.urloUrlSelector);
+  // const gistUrl = useSelector((state: State) => state.output.gist.url);
+  // const permalink = useSelector(selectors.permalinkSelector);
+  // const urloUrl = useSelector(selectors.urloUrlSelector);
 
   return (
     <Fragment>
-      <Copied href={permalink}>Permalink to the playground</Copied>
-      <Copied href={gistUrl}>Direct link to the gist</Copied>
+      {/*<Copied href={permalink}>Permalink to the playground</Copied>*/}
```

Comment:
[question]: Is this disabled because the local storage does not take precedent?

Reply:
I commented this out because we don't currently support the GitHub Gist integration that the playground offers. I do think this would probably just mean making a GH token and passing it through upon deployment, but we're not doing that. So, I'd rather remove these links than keep them broken. (You can see this in our current playground - try sharing via gist.)